_includes/event-post-macros.html

```diff
@@ -0,0 +1,245 @@
+    </time>
+{% endmacro %}
+
+
+
+{# ==========================================================================
+
+   event_location_image()
+
+   ==========================================================================
+
+   Description:
+
+   Create HTML image when given:
+
+   post: A post from a query result.
+
+   options (optional): An object to use for customizing a event location image using
+                       Google Maps.
+
+   options.class:    A class name to be applied to the img element.
+
+   options.location: A location as either a comma-separated {latitude,longitude} 
+                     pair (e.g. "40.714728,-73.998672") or a string address 
+                     (e.g. "city hall, new york, ny"). 
+
+   options.scale:    Affects the number of pixels that are returned.    
+                     Accepted values are 1, 2 and 4.    
+
+   options.size:     Defines the rectangular dimensions of the map image. 
+                     {horizontal_value}x{vertical_value} (e.g. 500x400)
+
+   options.zoom:     Determines the magnification level of the map. 
+
+      
+
+   ========================================================================== #}
+
+{% macro event_location_image(post, options) %}
+
+    {%- set location = (post.location or 'Washington DC') -%}
+    {%- set scale = ('&scale=' + (options.scale or '1')) -%}
+    {%- set size = ('&size=' + (options.size or '276x155')) -%}
+    {%- set zoom = ('&zoom=' + (options.zoom or '12')) -%}
+    {%- set url = 'https://maps.googleapis.com/maps/api/staticmap?center=' -%}
+    
+    <img class="{{ options.class }}" src="{{ url ~ location ~ size ~ zoom ~ scale }}" alt="Google Maps image of {{ location }}">
+{% endmacro %}
+
+
+  
+{# ==========================================================================
+
+   event_meta()
+
+   ==========================================================================
+
+   Description:
+
+   Create HTML event meta markup when given:
+
+   post: A post from a query result.
+
+   ========================================================================== #}
+
+{% macro event_meta(post) %}
+    <div class="event-meta h-event">
+        {{ event_meta_address(post) }}
+        {{ event_meta_datetime(post.date)  }}
+    </div>
+{% endmacro %}
+
+
+
+{# ==========================================================================
+
+   event_meta_address()
+
+   ==========================================================================
+
+   Description:
+
+   Create HTML event meta address markup when given:
+
+   date: A post date from a query result.
+
+   ========================================================================== #}
+
+{% macro event_meta_address(post) %}
+    {%- set city = (post.location or 'Washington DC') -%}    
+    {%- set state = (post.state or '') -%}
+    {%- set venue = (post.venue or 'CFPB') -%}
+    <p class="event-meta_address h-adr">
```

Comment:
Sounds like it. If it is the case I'll create a JIRA task for it and assign it to myself.